get-started/versioning-availability.md

```diff
@@ -7,208 +7,53 @@ products:
-| [Elasticsearch Curator](curator://reference/index.md) | 8.0.0 and later |
-| [Elastic Cloud Control (ECCTL)](ecctl://release-notes/index.md) | 1.14.0 and later |
-| [Elastic Serverless Forwarder for AWS](elastic-serverless-forwarder://reference/index.md) | 1.20.1 and later |
-| [Elastic integrations](https://www.elastic.co/docs/reference/integrations/all_integrations) | All versions |
-| [Search UI JavaScript library](search-ui://reference/index.md) | 1.24.0 and later |
-
-### APM agent and tool versions
-
-| Product | Version |
-| --- | --- |
-| [Elastic Distribution of OpenTelemetry Android](apm-agent-android://release-notes/index.md) | 0.1.0 and later |
-| [Elastic Distribution of OpenTelemetry iOS](apm-agent-ios://release-notes/index.md) | 1.0.0 and later |
-| [Elastic Distribution of OpenTelemetry Java](elastic-otel-java://release-notes/index.md) | 1.0.0 and later |
-| [Elastic Distribution of OpenTelemetry .NET](elastic-otel-dotnet://release-notes/index.md) | 1.0.0 and later |
-| [Elastic Distribution of OpenTelemetry Node.js](elastic-otel-node://release-notes/index.md) | 0.1.0 and later |
-| [Elastic Distribution of OpenTelemetry Python](elastic-otel-python://release-notes/index.md) | 0.1.0 and later |
-| [Elastic Distribution of OpenTelemetry PHP](elastic-otel-php://release-notes/index.md) | 0.1.0 and later |
-| [Elastic APM .NET Agent](apm-agent-dotnet://release-notes/index.md) | 1.0.0 and later |
-| [Elastic APM Go Agent](apm-agent-go://release-notes/index.md) | 2.0.0 and later |
-| [Elastic APM Java Agent](apm-agent-java://release-notes/index.md) | 1.0.0 and later |
-| [Elastic APM Node.js Agent](apm-agent-nodejs://release-notes/index.md) | 4.0.0 and later |
-| [Elastic APM PHP Agent](apm-agent-php://release-notes/index.md) | 1.0.0 and later |
-| [Elastic APM Python Agent](apm-agent-python://release-notes/index.md) | 6.0.0 and later |
-| [Elastic APM Ruby Agent](apm-agent-ruby://release-notes/index.md) | 4.0.0 and later |
-| [Elastic APM Real User Monitoring JavaScript Agent](apm-agent-rum-js://release-notes/index.md) | 5.0.0 and later |
-| [Elastic APM AWS Lambda extension](apm-aws-lambda://release-notes/release-notes.md) | 1.0.0 and later |
-| [Elastic APM Attacher for Kubernetes](apm-k8s-attacher://reference/index.md) | 1.1.3 |
-
-
-
-
+| ----- | ----- |
+| Elastic Stack versions | [{{stack}}](/get-started/the-stack.md) version 9.0.0 and later, including 9.1.0 |
+| Deployment types | [{{serverless-full}}](/deploy-manage/deploy/elastic-cloud/serverless.md), [{{ech}}](/deploy-manage/deploy/elastic-cloud/cloud-hosted.md), [{{ece}}](/deploy-manage/deploy/cloud-enterprise.md), [{{eck}}](/deploy-manage/deploy/cloud-on-k8s.md), and [Self-managed deployments](/deploy-manage/deploy/self-managed.md) |
+| Deployment versions | [{{ece}}](/deploy-manage/deploy/cloud-enterprise.md) 4.0.0 and later, [{{eck}}](/deploy-manage/deploy/cloud-on-k8s.md) 3.0.0 and later |
+| Serverless project types | {{es}}, {{observability}}, and {{elastic-sec}} |
```

Comment:
```suggestion
| Serverless project types | {{es}}, Elastic {{observability}}, and {{elastic-sec}} |
```
We should rather fix (and align...) the variables but would need to make sure we're not creating anything weird before :D